distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Link } from 'react-router';
+import axios from 'axios';
+
+
+import MG from 'metrics-graphics';
+import * as metricApi from '../../api/metric-api';
+
+
+function formatData(item) {
+  var result = [];
+  var data = item.points;
+
+  for (let i = 0; i < data.length; i++) {
+    result.push({
+      x: data[i]['refRank'],
+      y: data[i]['c'] * 100,
+      label: data[i]['b']
+    });
+  }
+  return result;
+}
+
+function generateChart(name, chart, width, height) {
+  var refLabels = {};
+  var formattedData = formatData(chart);
+
+  formattedData.map(chartItem => {
+    refLabels['' + chartItem.x] = chartItem.label;
```

Comment:
Nope it's simply the other syntax to specify a custom key to an object. In this case we have it as int by coincidence but it could be a string for binned metrics later.